tests/integration/test_storage.sh

```diff
@@ -35,9 +35,11 @@ setup () {
     # Perform setup routines here.
     copy /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage-setup.atomic-tests-backup
     TEST_DEV_1=/dev/vdb
+    findmnt -o SOURCE
     if findmnt -o SOURCE | grep "^$TEST_DEV_1"; then
-        findmnt -o SOURCE | grep "^$TEST_DEV_1" | uniq | xargs umount
+        exit 77
```

Comment:
Can you expand on this? For some reason, on Fedora, `/dev/vdb1` gets auto-mounted, but not on CentOS. So this is making the test get skipped on Fedora. What happens if you just let it umount and wipe the partition?

Reply:
I've tried several times to trigger this test today and most of the time it failed (on the `pvmove`), it looks like it is automatically remounted (just my guess), it passed a few times but it failed in all the other cases.  Given that it has been a flake for the past months on Fedora, I think it is safer to skip it if cannot be trusted at all.